src/lib/Utils.ts

```diff
@@ -273,5 +273,17 @@ export function addImport(file: BrsFile, pkgPath: string) {
         // eslint-disable-next-line @typescript-eslint/no-unsafe-argument
         file.attachDependencyGraph(dg);
     }
+}
 
+/**
+ * Handles replacing windows `\` with `/`. Also handles v0->v1 compatibility by prepending `pkg:/` if missing
+ * @param pkgPath the full pkgPath of a file (with or without `pkg:/`). This MUST NOT be a relative path
+ */
+export function sanitizePkgPath(pkgPath: string) {
```

Comment:
bsc@v1 stores the full `pkg:/...` path in the `file.pkgPath` variable, but bsc@v0 stores it without the `pkg:/` and with os-specific path separators (idk why I did it this way....but oh well). 

This function handles sanitizing the pkgPath to always produce a proper `pkg:/` path with the correct `/` paths. However, you can't use this function when you're going to compare `file.pkgPath`, so only use this function right before writing pkgPaths to Roku files.